modules/page-memory/src/testFixtures/java/org/apache/ignite/internal/pagememory/util/PageLockListener.java

```diff
@@ -27,60 +27,46 @@ public interface PageLockListener extends ManuallyCloseable {
     /**
      * Callback that's called before write lock acquiring.
      *
-     * @param groupId Group ID.
-     * @param pageId Page ID.
-     * @param page Page pointer.
+     * @param lock Lock pointer.
      */
-    void onBeforeWriteLock(int groupId, long pageId, long page);
+    void onBeforeWriteLock(long lock);
```

Comment:
Please rename to lockPtr below and heirs.

Reply:
We usually call them "children". Here I will do a rename, because it's a "new" class